### DISCUSSION TITLE
Error - Inisiasi object medel menggunakan fungsi Prophet

### DISCUSSION THREAD

**Discussion by rastitiratnasari** [teamalgoritma/community#839] [Q&A] [answered]
![Screenshot (26)](https://github.com/teamalgoritma/community/assets/150925524/70cbf9f0-b0e1-42b0-acec-11991546068d)

Bagaimana mengatasi error diatas
Data sdh dalam bentuk 'ds' dan 'y'

**Comment by saskia-dwi-ulfah** [comment, marked as answer]
Deak Bu @rastitiratnasari 
Dari gambar yang ibu lampirkan, errornya terjadi karena ibu memasukkan data sebagai string (menggunakan tanda petik). Tanda petiknya dihapus saja dan kodenya akan menjadi seperti berikut: `model.fit(daily_total_revenue)`.

Semoga jawaban ini dapat membantu.

Best regards,
Team Algoritma

**Reply by rastitiratnasari** [reply]
Dear Mba Ia,
well noted mba terimakasih banyak.
